common/components/ConfirmationDialog/ConfirmationDialog.tsx

```diff
@@ -30,7 +32,7 @@ const ConfirmationDialog = ({
         <Button variant="secondary" onClick={onCancel}>
           {cancel}
         </Button>
-        <Button variant="default" onClick={onConfirm}>
+        <Button variant="default" disabled={disableConfirm} onClick={onConfirm}>
```

Comment:
Suggestion add a tooltip for the user why the button is disabled for example "Please make changes " to indicate what the user is doing wrong or indication that the text has been change like "Changes have been made"

Reply:
Tooltip is now added. There are two different versions for "Choose a collection". If there is no preselected collection (the word is not added to a collection prior opening the dialog) then the tooltip says "Choose a collection". But if there is any preselected collection the tooltip says "The word is already added to the selected collections".

<img width="443" alt="Skjermbilde 2024-09-16 kl  17 39 00" src="https://github.com/user-attachments/assets/c44cc292-081c-41e4-a236-dcb4186589c3">
<img width="427" alt="Skjermbilde 2024-09-16 kl  17 39 12" src="https://github.com/user-attachments/assets/94e8705f-3b53-400d-8c11-a43896739857">